src/content/en/2024/sustainability.md

```diff
@@ -2,17 +2,1265 @@
+
+For <a hreflang="en" href="https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs">optimal favicon</a> sustainability:
+1. SVG: The ideal choice. SVGs are lightweight, scalable, and eliminate the need for multiple sizes, significantly reducing data transfer and storage needs.
+2. Optimized PNG: A well-optimized PNG (180x180 pixels for iOS in the base directory named apple-touch-icon.png) can be a good compromise between file size and broad compatibility. Note the filename and location requirements as browsers will seek this (based on the meta tag if used) if no SVG is provided.
+3. A favicon.ico with sizes 32x32 and 16x16 for compatibility purposes (older browsers will seek this within your base directory so having the file will reduce errors).
+These formats, when properly implemented, minimize data transfer, reduce storage requirements, and lower energy consumption across the web ecosystem. Always ensure your chosen format is properly optimized for the best sustainability impact.
+
+{{ figure_markup(
+  image="favicon-usage.png",
+  caption="Favicon usage",
+  description="A bar chart showing that png is the format used for favicons on 41.9% of pages on both desktop and mobile. ico format is used by 28.3% on desktop and 27.2% on mobile. Favicon is missing in 17.2% on desktop and 17.9% on mobile. jpg is the format used by 6.2% on desktop and 6.5% on mobile. Svg is used by 1.4% on both desktop and mobile. NO_EXTENSION, webp, NO_DATA, gif, jpeg and NO_HREF repersent less than 1% each, on desktop and mobile.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRQLDsXdZj62xe68w716gen0rQvuuGhXPAOSwdWwYjSBZf9BgJgEb-dp1Z_jB_Lp5YMsfH0FiNKwzDb/pubchart?oid=1102352877&format=interactive",
+  sheets_gid="1918305009",
+  sql_file="favicons.sql"
+) }}
+
+When we compare the 2024 data to <a hreflang="en" href="../2021/markup#favicons">2021 data</a> (since there was no favicon data since the <a hreflang="en" href="../2021/markup#favicons">2021 Markup chapter</a>), we can say that the changes from 2021 to 2024 indicate a positive trend towards more sustainable favicon practices. The shift from ICO to more efficient formats like PNG, SVG, and WebP suggests improved awareness of file size and performance impacts. Also, the reduction in missing icons demonstrates better attention to detail, reducing unnecessary server requests. Finally, the growth in SVG and WebP usage, while still small, represents a move towards more sustainable, scalable formats.
+
+While there's still room for improvement, particularly in further adopting highly efficient formats like SVG and WebP, the overall trend suggests that developers are increasingly considering the sustainability implications of even small elements like favicons.
+
+For further information, refer to:
+- The WSGs 1.0 - SC 3.18 and 4.4.
+
+### Optimizing external content
+
+{# TODO - add intro paragraph #}
```

Comment:
@burakguneli I think this is what @lebreRafael wanted you to help with?

Reply:
@lebreRafael I added the intro paragraph to the google docs: https://docs.google.com/document/d/1AB_e7VaHPC307NycTXxwndOTgbFJld2_RqWm8abps1w/edit?disco=AAABXuHfL4M

I don't know much about markdown so I didn't want to touch it. Would it be okay for you to review it and add it to the PR if everything looks good?